docs/migration.md

```diff
@@ -0,0 +1,35 @@
+---
+layout: default
+title: Migrate to Version 7
+permalink: /migration/
+nav_order: 2
+---
+
+# Migrate from Version 6.x to Version 7.x
+
+Version 7.x introduces a redesigned architecture based on TypeScript.
+We aimed for seamless migration from Version 6.x to Version 7.x for most users.
+However, we also use this major version increment to get rid of some outdated concepts and features.
+On the other hand we also introduce a new configuration concepts, which allow full customization of the generated fields and field contents.
```

Comment:
```suggestion
This enables the customization of generated fields and field contents and get rid of some obsolete features.
```
A major version includes always breaking changes ... I would concentrate on the positive